### DISCUSSION TITLE
'primereact/calendar' has no exported member named { CalendarChangeEvent }

### DISCUSSION THREAD

**Discussion by iceddrop** [primefaces/community#2706] [PrimeReact] [answered]
`import {  Calendar, CalendarChangeEvent } from 'primereact/calendar';`

This import (CalendarChangeEvent) seems to be deprecated in the current version of prime react , and the update is yet to reflect in the documentation

what i am trying to use it for:
`  <Calendar value={options.value} onChange={(e: CalendarChangeEvent) => options.filterCallback(e.value, options.index)} dateFormat="mm/dd/yy" placeholder="mm/dd/yyyy" mask="99/99/9999" />`.

You will find this code on this [page](https://primereact.org/datatable/) , under Advanced filtering ( Typescript )

What is the new update pls?

**Comment by qwadrox** [comment, marked as answer]
@iceddrop 
You're right. I've created an issue.

https://github.com/primefaces/primereact/issues/7218

**Reply by melloware** [reply]
Fixed for next release
